desugreet/bot.py

```diff
@@ -53,10 +53,14 @@ async def on_member_update(member_before, member_after):
             await client.get_channel(log_channel_id).send("{0.mention} (closed DM) isch cho".format(member_after))
         else:
             await client.get_channel(log_channel_id).send("{0.mention} isch cho".format(member_after))
-        # clean messages in entrance_track_buffer
+        # delete messages in entrance_track_buffer
         if member_after in entrance_track_buffer.keys():
             for msg in entrance_track_buffer[member_after]:
-                await msg.delete()
+                try:
+                    await msg.delete()
+                except discord.errors.NotFound:
+                    pass
```

Comment:
maybe log the user-id and message-id here?

Reply:
or at least something like `failed to delete messages in entrance`, to have some sort of feedback that something unexpected happened

Reply:
I do think we need to find a way to log general activities of the bot itself and maybe even display them per website so we can get a quick overview when something is up.